python/ccf/clients.py

```diff
@@ -389,6 +426,9 @@ def request(self, request, signed=False, timeout=DEFAULT_REQUEST_TIMEOUT_SEC):
                 )
             except Exception as exc:
                 raise CCFConnectionException from exc
+
+        assert self.ws is not None
```

Comment:
Is this necessary? Can create_connection() fail but not raise?

Reply:
This seems necessary for `mypy` to be happy. I guess that's because `websocket` doesn't have type annotations. Without it: 
```
$ mypy ../python/
../python/ccf/clients.py:439: error: "None" has no attribute "send_frame"
../python/ccf/clients.py:440: error: "None" has no attribute "recv_frame"
```

Reply:
That's unfortunate...